### ISSUE TITLE
Formula not reported in excel 2007

### ISSUE THREAD

**Issue by feerrenrut** [nvaccess/nvda#9431]
### Steps to reproduce:
1. Create a new sheet in MS excel 2007
1. Add a simple formula to a cell and press enter
1. Use arrow keys to return to the cell with the formula

### Actual behavior:
The result of the formula is reported (calculated value). "has formula" is not announced.
Pressing F2 allows us to confirm the presence of the formula.

### Expected behavior:
"has formula" should be announced.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
`alpha-16976,7fc6c71f`

#### Windows version:
Microsoft Windows
Version 1803 (OS Build 17134.648)

#### Name and version of other software in use when reproducing the issue:
Microsoft Office Excel 2007 (12.0.6787.5000) SP3 MSO (12.0.6785.5000) 

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA?
2018.4 works as expected.

**Comment by feerrenrut** [comment]
Some alternatives that can be used to work around this issue can be found in https://github.com/nvaccess/nvda/issues/7899. Specifically, pressing ```control + ` (accent grave)``` in excel will toggle between showing formulae in (all) cells rather than the result.

**Comment by michaelDCurran** [comment]
Does inserting a comment on the cell, cause NVDA to announce "has comment"? If not, I am guessing this is a broader problem where NVDA is not able to inject into Excel at all.

**Comment by feerrenrut** [comment]
No, "has comment" is also not reported. The NVDA log contains the following:

The most relevant thing in the log seems to be (there a few similar such errors):
```
DEBUGWARNING - eventHandler.executeEvent (14:31:39.773):
error executing event: typedCharacter on <NVDAObjects.window.excel.ExcelCell object at 0x0563C6B0> with extra args of {'ch': u'l'}
Traceback (most recent call last):
  File "eventHandler.pyc", line 155, in executeEvent
  File "eventHandler.pyc", line 92, in __init__
  File "eventHandler.pyc", line 100, in next
  File "NVDAObjects\window\excel.pyc", line 1342, in event_typedCharacter
  File "baseObject.pyc", line 47, in __get__
  File "baseObject.pyc", line 147, in _getPropertyViaCache
  File "NVDAObjects\window\excel.pyc", line 871, in _get_states
  File "comtypesMonkeyPatches.pyc", line 35, in new__getattr__
  File "comtypes\client\lazybind.pyc", line 168, in __getattr__
  File "comtypes\automation.pyc", line 729, in _invoke
COMError: (-2147418111, 'Call was rejected by callee.', (None, None, None, 0, None)
```

**Comment by fisher729** [comment]
Hi.

NVDA no longer causes Excel 2007 to crash or refuses to report if a cell has a formular.
Should be corrected to: NVDA no longer causes Excel 2007 to crash or refuses to report if a cell has a formula. (no R)

**Comment by Adriani90** [comment]
I think this issue is solved in last NVDA Alpha. Right?

**Comment by Adriani90** [comment]
@feerrenrut is this solved in NVDA 2019.1.1?

**Comment by feerrenrut** [comment]
Fixed in #9456
